### ISSUE TITLE
Fluids don't get pumped if there is an IC2 iron scaffold inside of them

### ISSUE THREAD

**Issue by witepah** [xJon/Tekkit-2#262]
### Was this bug reported before?

- [X] No, it was not.

### Please describe the issue you are experiencing

As the title says, the IC2 scaffold block interacts wierdly with the pump from BuildCraft. I noticed this while trying to pump oil but the pump only pumps out the big pillar in the middle untill it reaches ground level and then it stops. Removing the scaffold from the oil solves this. How to reproduce: put an iron scaffold block or more inside an oil puddle and start pumping it out. When the oil level reaches the scaffold it will stop pumping.
![image](https://github.com/user-attachments/assets/2664d778-e9f6-4d9f-b740-064e14d33052)
![image](https://github.com/user-attachments/assets/4d03a092-b7e7-4f17-83c6-cfc53bf1bb82)

**Comment by xJon** [comment]
Can you go to `\tekkit-2\config\fluidlogged_api.cfg`, delete these 2 lines, and try again? The blocks might need placing again.
```
	{blockId:"ic2:blockironscaffold", canFluidFlow:true},
	{blockId:"ic2:blockscaffold", canFluidFlow:true},
```

**Comment by witepah** [comment]
I tried it on a test world and by removing those lines it works as expected. The oil pumping doesn't bug out anymore. 
![image](https://github.com/user-attachments/assets/5ed9fb55-10f3-4982-a302-52cbae992bb7)

**Comment by xJon** [comment]
Resolved in `1.2.4`.
